mdaviz/mda_file_table_model.py

```diff
@@ -0,0 +1,219 @@
+"""
```

Comment:
Thanks for showing a _simple_ way to use checkboxes with a table model/view.  The online versions all use a delegate.  This use is not nearly so complicated to need a delegate.